packages/hardhat-core/src/internal/core/config/config-validation.ts

```diff
@@ -573,3 +574,17 @@ export function getValidationErrors(config: any): string[] {
   const ioTsErrors = DotPathReporter.report(result);
   return [...errors, ...ioTsErrors];
 }
+
+export function validateResolvedConfig(resolvedConfig: HardhatConfig) {
+  console.log(resolvedConfig.solidity.compilers);
```

Comment:
I guess you added this for debugging? We should remove it.

Reply:
Yes, My bad forgot to remove